cardano-constitution/src/Cardano/Constitution/Validator/Static.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE TemplateHaskell   #-}
+-- Following is for tx compilation
+{-# LANGUAGE DataKinds         #-}
+{-# OPTIONS_GHC -fplugin PlutusTx.Plugin #-}
+{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:target-version=1.1.0 #-}
+module Cardano.Constitution.Validator.Static
+    ( defaultConstitutionValidator
+    , defaultConstitutionValidatorUntyped
+    , defaultConstitutionCode
+    ) where
+
+import Cardano.Constitution.Config.Default
+import Cardano.Constitution.Predicate
+import Cardano.Constitution.ProtocolParam
+import Cardano.Constitution.Validator.Common
+import Cardano.Constitution.Validator.Static.TH
+import PlutusLedgerApi.V3 as V3
+import PlutusTx as Tx
+import PlutusTx.Prelude as Tx
+
+defaultConstitutionValidator :: () -> V3.ScriptContext -> Bool
+defaultConstitutionValidator () =
+    \case
+        -- NOTE: this script will ignore (and thus succeed) on unknown indices inside cparams;
+        -- is it then the responsibility of the ledger to first sanitize cparams?
+        -- NOTE: this script will succeed on an empty cparams;
+        -- is it then the responsibility of the cardano-ledger to first sanitize cparams?
+        GetChangedParamsMap cparams ->
+            -- I could use `Tx.Bool.and` here, but probably is more costly
+                  $$(satisfies TxFeePerByte MinValue defaultConstitutionConfig) cparams
```

Comment:
If we're only ever going to be checking that a value lies in some interval (I don't know if that's actually the case) then we could probably just have a single check for each parameter instead of having to check the minimum and maximum values separately.

Reply:
@zliu41 proposed the same thing, but I don't know if in the future there will be differenet kind of rails and maybe unbounded (no min or maxbound). Not hardcoding them  as (<= and >=) allows the flexibility to 1) add more predicates in the future, and 2) even leaving some parameters (min/max-) unbounded. 
The cost of not hardcoding them I think should not be measurable (at least for the Static version). The static version short of "hardcodes" them at compile-time, while still having the flexibility (1) and (2)